keep/contextmanager/contextmanager.py

```diff
@@ -28,6 +28,13 @@ def __init__(self):
         self.foreach_context = {}
         self.click_context = click.get_current_context()
 
+    # TODO - If we want to support multiple alerts at once we need to change this
```

Comment:
wdym?

Reply:
that its not "thread safe" so if someday we will want "parallel" alerts, we will need to override this.